docs/sql-reference/10-sql-commands/00-ddl/02-user/23-grant-ownership.md

```diff
@@ -0,0 +1,43 @@
+---
+title: GRANT OWNERSHIP
+sidebar_position: 13
+---
+import FunctionDescription from '@site/src/components/FunctionDescription';
+
+<FunctionDescription description="Introduced or updated: v1.2.275"/>
+
+Grants ownership of a specific database objects, such as tables, stages, or user-defined functions, to a specified role. 
```

Comment:
Better to add what's OWNERSHIP.

Reply:
Yes. I will explain what's OWNERSHIP in the Security Guide and reference this page to the guides.